lib/rules/no-curly-component-invocation.js

```diff
@@ -126,6 +128,13 @@ module.exports = class NoCurlyComponentInvocation extends Rule {
             return;
           }
 
+          if (BUILT_INS.includes(name)) {
+            // {{debugger}}
+            // {{outlet}}
+            // {{yield}}
```

Comment:
this looks incorrect. this code path is only about invocationa with named arguments.

Reply:
ah, you mean examples in comments? will fix it, because it just copy-pasted from namedless example.

and without this block itself test for <https://github.com/ember-template-lint/ember-template-lint/pull/1136/files#diff-37bf2986269b4e055b1de08cc17fa5e1R44> failing